packages/astro-ghostcms/src/api/ghostAPI.test.ts

```diff
@@ -14,21 +18,25 @@ describe("content-api", () => {
 
 	test("content-api shouldn't instantiate with an incorrect url", () => {
 		assert.throws(() => {
-			const api = new TS_API("ghost.org", "59d4bf56c73c04a18c867dc3ba", "v5.0");
+			const api = new TSGhostContentAPI(
+				"ghost.org",
+				"59d4bf56c73c04a18c867dc3ba",
```

Comment:
Are these live keys?? Might not want to commit these!

Reply:
These are Ghosts public demo keys that ghost provides for testing purposes